versioned_docs/version-1.2/community/reporting_security_concerns.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Reporting Security Concerns
+sidebar_position: 50
+---
+
+# Security Policy
+
+Thank you for taking the time to report a security vulnerability. 
+We would like to investigate every report thoroughly.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability, please follow these steps:
+
+**Step 1**
+
+Navigate to the appropriate reporsitory.
+
+**Step 2**
+
+Click on `Security` and then `Report a vulnerability`
+
+![Screenshot of how to report a vulnerability](/img/reporting_a_security_concern.png)
```

Comment:
This will make `1.2` use the picture of `next` release, is it intended?

Reply:
Let's ignore those comments and focus on which is the optimal solution for versioning.